src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -92,7 +89,7 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
 
         initializeUiComponents()
 
-        displayTreeAndDescriptionPanels()
+        chooseMainPanelToDisplay()
```

Comment:
good catch and re-use `chooseMainPanelToDisplay()` here and in `checkCliExistsFinished`